apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/index.tsx

```diff
@@ -246,19 +248,30 @@ const SyncOnboardingManual = ({ deviceModelId: strDeviceModelId }: SyncOnboardin
       return;
     }
 
+    // case DeviceOnboardingStep.SafetyWarning not handled so the previous step (new seed, restore, recover) is kept
     switch (deviceOnboardingState?.currentOnboardingStep) {
       case DeviceOnboardingStep.SetupChoice:
-      case DeviceOnboardingStep.SafetyWarning:
         setStepKey(StepKey.Seed);
+        setSeedPathStatus("choice_new_or_restore");
         break;
       case DeviceOnboardingStep.NewDevice:
       case DeviceOnboardingStep.NewDeviceConfirming:
         setShouldRestoreApps(false);
         setStepKey(StepKey.Seed);
+        setSeedPathStatus("new_seed");
+        break;
+      case DeviceOnboardingStep.SetupChoiceRestore:
+        setStepKey(StepKey.Seed);
+        setSeedPathStatus("choice_restore_direct_or_recover");
         break;
       case DeviceOnboardingStep.RestoreSeed:
         setShouldRestoreApps(true);
         setStepKey(StepKey.Seed);
+        setSeedPathStatus("restore_seed");
+        break;
+      case DeviceOnboardingStep.RecoverRestore:
+        setStepKey(StepKey.Seed);
```

Comment:
no `setShouldRestoreApps(true)` here?

Reply:
because it's the Recover path, there is no app restoring (i think). If there is, we can update it quickly ;)